### DISCUSSION TITLE
State token expires too quickly

### DISCUSSION THREAD

**Discussion by mat02** [lowdefy/lowdefy#937] [Questions and Help]
#### Describe the bug
When app is used on shared computer (i.e. browser in kiosk mode), user may log in after some inactivity period, which is longer than state token expiration time. User is then showed "Error 500 AuthenticationError: Token expired." and has to click on "Return to home page" link. There is no way to extend that timeout.

#### To Reproduce
1. Create page with OpenID authentication.
2. Open Lowdefy app and wait for redirection to login page.
3. Wait for more than 5 minutes.
4. Login

#### Screenshots
![image](https://user-images.githubusercontent.com/1629659/140488637-6bf262df-7bb6-4b38-bea4-b9094b3184f7.png)

#### Lowdefy Version
3.22.0

**Comment by Gervwyk** [comment]
Sorry. I transferred the issue too quickly. This should be a feature issue, not a discussion, and I could not reopen to issue for some reason. New issue is #938 
Thanks @mat02
